### PR TITLE
[new] Added url.js

### PR DESCRIPTION
**Pull request by IonicaBizau**
Added the url.js library: https://github.com/jillix/url.js

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Thanks.

**Comment by IonicaBizau** [comment]
We (@jillix) thank you! :smile:

**Comment by PeterDaveHello** [comment]
np

**Comment by IonicaBizau** [comment]
@PeterDaveHello I forgot to add the `autoupdate` field in package.json. Would't it be better to modify the scripts to get updates from npm? Otherwise, we would have some redundant information...

File: ajax/libs/urljs/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "manipulate",
+    "browser"
+  ],
+  "author": "jillix <contact@jillix.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jillix/url.js/issues"
+  },
+  "homepage": "https://github.com/jillix/url.js",
+  "npmName": "urljs",
```
Comment (PeterDaveHello) [inline ajax/libs/urljs/package.json]: It's already set to update from npm.

Comment (IonicaBizau) [inline ajax/libs/urljs/package.json]: What about if I don't provide that? Shouldn't it update automatically using `name` field?

Comment (PeterDaveHello) [inline ajax/libs/urljs/package.json]: You meant you don't provide npm package? It uses npmName not name field.

Comment (IonicaBizau) [inline ajax/libs/urljs/package.json]: I mean, why not to use the `name` field instead of `npmName`?

Comment (PeterDaveHello) [inline ajax/libs/urljs/package.json]: We can not make sure they are the same and if npm package exist.

Comment (IonicaBizau) [inline ajax/libs/urljs/package.json]: Ah, I see. Thanks! :+1:

**Pull request merged**